main.go

```diff
@@ -16,6 +16,7 @@ func main() {
 	GetCpuUsage()
 	GetRamUsage()
 	GetDiskUsage()
+	ResultPrinter("Public IPv4 Address: ",GetPublicIPAddress())
```

Comment:
Print the Result in the function call itself instead of returning a string, this is to ensure decoupled functions performing their own operations, hence kindly print the result in function itself. I should just need to call the function in ```main.go```

Reply:
Okay, I will make the desired changes